website/docs/language/state/purpose.mdx

```diff
@@ -59,7 +59,7 @@ of dependencies within the state. Now Terraform can still determine the correct
 order for destruction from the state when you delete one or more items from
 the configuration.
 
-One way to avoid this would be for Terraform to know a required ordering
+Another way would be for Terraform to know a required ordering
```

Comment:
Hello! Thank you so much for your suggestion here, I agree this language is definitely a bit vague. 

I do think we could make this _even_ more explicit by saying something like:
"Terraform could take another approach to dependency order by using an underlying hierarchy of order between resource types."

If you like that, then my suggestion would be:
```suggestion
Terraform could take another approach to dependency order by using an underlying hierarchy of order
```

Let me know what you think!

Reply:
Seems good